lua/packer.lua

```diff
@@ -263,20 +263,22 @@ local function args_or_all(...) return util.nonempty_or({...}, vim.tbl_keys(plug
 -- managed plugins.
 -- Installs missing plugins, then updates helptags and rplugins
 packer.install = function(...)
-  local install_plugins
-  if ... then
-    install_plugins = {...}
-  else
-    install_plugins = plugin_utils.find_missing_plugins(plugins)
-  end
+  local args = ...
```

Comment:
I believe this only keeps the first argument passed to `packer.install()`, is this intended?

Reply:
Nope if that's the case then definitely not the intention, this is more to get past the fact that I can't use `varargs` directly since where they are needed is now within a new function scope although, I could just collect them into a table and unpack that later e.g. `local args = {...}`